conformance/connectivity.go

```diff
@@ -69,4 +70,52 @@ var _ = Describe("Connectivity to remote services", func() {
 			})
 		})
 	})
+
+	Context("Connectivity to a ClusterIP service existing in two clusters but exported from one", func() {
+		BeforeEach(func() {
+			requireTwoClusters()
+		})
+
+		JustBeforeEach(func() {
+			t.deployHelloService(&clients[1], newHelloService())
+		})
+
+		It("should only access the exporting cluster", Label(OptionalLabel, ConnectivityLabel, ClusterIPLabel), func() {
+			AddReportEntry(SpecRefReportEntry, "https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/1645-multi-cluster-services-api/README.md#exporting-services")
+
+			By(fmt.Sprintf("Exporting the service on cluster %q", clients[0].name))
+
+			t.createServiceExport(&clients[0])
+
+			By(fmt.Sprintf("Awaiting service deployment pod IP on cluster %q", clients[0].name))
+
+			servicePodIP := ""
+
+			Eventually(func() string {
+				pods, err := clients[0].k8s.CoreV1().Pods(t.namespace).List(context.TODO(), metav1.ListOptions{
+					LabelSelector: metav1.FormatLabelSelector(newHelloDeployment().Spec.Selector),
+				})
+				Expect(err).ToNot(HaveOccurred())
+
+				if len(pods.Items) > 0 {
+					servicePodIP = pods.Items[0].Status.PodIP
+				}
+
+				return servicePodIP
```

Comment:
Not sure I am following correctly this test, why are you fetching the pod IP? I am not entirely sure how to test this case shouldn't this get the IP from the ServiceImport somehow?

Reply:
The service's backend deployment returns the [pod IP](https://github.com/kubernetes-sigs/mcs-api/blob/e2cb6dc0c7538264e2e57cbbe28172ee2dccbb97/conformance/k8s_objects.go#L78) in response to requests. So I check for the pod IP from the deployment on cluster 1 to ensure only this service is accessed.

Reply:
Ahh yes indeed I see it now!